### PR TITLE
More spam assets, keep some prices at global DB upgrade, decoder wording

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: rotkehlchen/assets/spam_assets.py [hunk 1]
```diff
@@ -248,15 +248,438 @@
...
+    string_to_evm_address('0x708B63545467a9bCFB67aF92299102c650E34a0e'): {
+        'name': 'Vital  Ethereum',
+        'symbol': 'VETH',
+        'decimals': 18,
+    },
+    string_to_evm_address('0x77FE30b2cf39245267C0a5084B66a560f1cF9E1f'): {
+        'name': 'Azbit',
+        'symbol': 'AZ',
+        'decimals': 18,
+    },
```
Comment (yabirgb) [inline rotkehlchen/assets/spam_assets.py:450]: lol, they got the same name as a real token

File: rotkehlchen/globaldb/upgrades/v2_v3.py [hunk 2]
```diff
@@ -328,12 +328,21 @@ def translate_assets_in_price_table(cursor: 'DBCursor') -> List[Tuple[str, str,
     Translate the asset ids in the price table.
 
     Also drop all non manually input asset prices since otherwise this upgrade
-    will take forever. A heavily used globaldb
+    will take forever for a heavily used globaldb
     """
-    # cursor.execute('DELETE from price_history WHERE source_type!="A"')
+    assets = (
+        'ETH',
+        'ETH2',
```
Comment (yabirgb) [inline rotkehlchen/globaldb/upgrades/v2_v3.py:335]: Maybe we should be storing the ETH2 price as ETH? Basically we are duplicating information here

Comment (LefterisJP) [reply rotkehlchen/globaldb/upgrades/v2_v3.py:335]: I think we maybe do so. But just putting it here just in case.

**Comment by LefterisJP** [comment]
Only failed test is opensea rate limit (note it fails ALL 3 flaky tries :( )

**Pull request merged**